### PR TITLE
Add hook to metabox after enable option

### PR DESCRIPTION
**Pull request by mintplugins**
#221

### CODE REVIEW

File: includes/admin/classes/EDD_C_List_Table.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ function column_default( $item, $column_name ) {
 				if ( 'percentage' == $type ) {
 					return $item[ $column_name ] . '%';
 				} else {
-					return edd_currency_filter( edd_sanitize_amount( $item[ $column_name ] ) );
+					return apply_filters( 'eddc_format_rate', edd_currency_filter( edd_sanitize_amount( $item[ $column_name ] ) ), $item[ $column_name ], $type );
```
Comment (pippinsplugins) [inline includes/admin/classes/EDD_C_List_Table.php]: @mintplugins could you turn this instead a standard helper function and the update the whole plugin to use it anywhere we output the commission amount?

Comment (mintplugins) [inline includes/admin/classes/EDD_C_List_Table.php]: For sure. I have called it eddc_format_rate and applied it everywhere I could find it now.

**Comment by mintplugins** [comment]
@pippinsplugins  I have called a helper function called eddc_format_rate which takes a rate and commission type and formats it for output. I also applied it everywhere I could find it now. I also tested each page where it was replaced and confirmed they all work as expected.

**Comment by mintplugins** [comment]
Places where the rate was changed:

- Shortcode [edd_commissions_overview]
- Dashboard Widget
- Customer tab
- Commissions list table
- Single Commissions view

**Comment by ghost1227** [comment]
Post-merge tests say that we didn't break anything with the 3.3.3 branch merge. This tests good for me! @pippinsplugins Any reason we can't close this out?

**Pull request merged**